libutils/file_lib.h

```diff
@@ -67,6 +67,16 @@ Writer *FileReadFromFd(int fd, size_t size_max, bool *truncated);
 
 bool FileCanOpen(const char *path, const char *modes);
 
+/**
+ * Returns true if a path (such as a file or a folder) exists
+ * Don't follow symlinks, don't check permissions, don't check if it's a file.
```

Comment:
This is very similar to `access(path, F_OK)`, the only difference is that `access()` follows symlinks. Maybe it could be mentioned here as an alternative for such cases?

Reply:
Yes, good idea.